temporal/api/cloud/identity/v1/message.proto

```diff
@@ -69,3 +69,27 @@ message User {
     // Will not be set if the user has never been modified.
     google.protobuf.Timestamp last_modified_time = 8;
 }
+
+message GroupSpec {
+    // The human-readable name of the group
```

Comment:
Isn't this more than just a human-readable name? Its the name of the external group, in this case a the name of the google workspace group.
Also i would add a comment stating mutable. I am assuming its not.